Community-Aktivitäten/README.md

```diff
@@ -165,6 +165,46 @@ Dauer: 2 Stunden (bei Bedarf auch gerne mehr)
 
 Ausstattung: großer Screen/Beamer, Internet
 
+## Building your career: Ask anything
+:us: :de:
+
+### Collaborators
+* [JUG Schweiz]: [Ixchel Ruiz]
+* [JUG Dortmund]: [Sandra Parsick]
+* [JUG Dortmund]: [Hendrik Ebbers]
+
+### Description:
+If you're looking for career advice in software development, a good strategy is to ask your seniors. They have valuable experience and can provide insights that will help guide your professional development. Their first-hand knowledge can give you a clear understanding of the industry and its requirements. Don't hesitate to join this event with any questions or concerns you may have. By seeking the guidance and advice of our guests, you'll be better equipped to make informed decisions about your career path.
+
+### Duration : 2 hr : 1 hr Tuesday & Wednesday during the conference ( 2 Days ) 
+
+### Beschreibung:
+If you're looking for career advice in software development, a good strategy is to ask your seniors. They have valuable experience and can provide insights that will help guide your professional development. Their first-hand knowledge can give you a clear understanding of the industry and its requirements. Don't hesitate to come to this event with any questions or concerns that you may have. By seeking the guidance and advice of our guests, you'll be better equipped to make informed decisions about your career path.
+
+### Dauer: 2 Stunden : 1 Std. Dienstag & Mittwoch während der Konferenz ( 2 Tage )
+
+### Equipment: Panel of 4 seats ( fishbowl format )
+
+## Work Life : Cultural Transformation with Diversity and Inclusion
```

Comment:
Is that a *second* activity? Then please separate it into its own draft, as we need to be able to accept / reject all activities independently (e. g. if not enough rooms are free). At least unless you think that a reject of *one* implies a reject of *both*. Thanks.

Reply:
@markg done and updated other comments!